FFmpeg.AutoGen.Example/FFmpegBinariesHelper.cs

```diff
@@ -1,25 +1,37 @@
 ﻿using System;
 using System.IO;
+using System.Runtime.InteropServices;
 
 namespace FFmpeg.AutoGen.Example
 {
     public class FFmpegBinariesHelper
     {
         internal static void RegisterFFmpegBinaries()
         {
-            var current = Environment.CurrentDirectory;
-            var probe = Path.Combine("FFmpeg", "bin", Environment.Is64BitProcess ? "x64" : "x86");
-            while (current != null)
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
-                var ffmpegBinaryPath = Path.Combine(current, probe);
-                if (Directory.Exists(ffmpegBinaryPath))
+                var current = Environment.CurrentDirectory;
+                var probe = Path.Combine("FFmpeg", "bin", Environment.Is64BitProcess ? "x64" : "x86");
+                while (current != null)
                 {
-                    Console.WriteLine($"FFmpeg binaries found in: {ffmpegBinaryPath}");
-                    ffmpeg.RootPath = ffmpegBinaryPath;
-                    return;
-                }
+                    var ffmpegBinaryPath = Path.Combine(current, probe);
+                    if (Directory.Exists(ffmpegBinaryPath))
+                    {
+                        Console.WriteLine($"FFmpeg binaries found in: {ffmpegBinaryPath}");
+                        ffmpeg.RootPath = ffmpegBinaryPath;
+                        return;
+                    }
 
-                current = Directory.GetParent(current)?.FullName;
+                    current = Directory.GetParent(current)?.FullName;
+                }
+            }
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            {
+                ffmpeg.RootPath = "/lib/x86_64-linux-gnu/";
```

Comment:
This works with wsl1

Reply:
I do have a feeling it can be improved and should be a part of app.config or even environment var

Reply:
My argument is that, as example project, it's better to work out of box instead of having to make code change or setup env var, but I can make the change if you insisit : )